src/components/Form.js

```diff
@@ -294,5 +295,6 @@ if (process.env.NODE_ENV !== "production") {
     transformErrors: PropTypes.func,
     safeRenderCompletion: PropTypes.bool,
     formContext: PropTypes.object,
+    localization: PropTypes.func,
```

Comment:
Why did you rename it from `localize` to `localization`? Just want to hear your thinking on this.